variables.tf

```diff
@@ -15,6 +15,12 @@ variable "groups" {
   description = "Set of group names to attach to the user"
 }
 
+variable "permissions_boundary" {
+  type        = string
+  default     = null
+  description = "The permissions boundary to set on the user"
```

Comment:
The ARN of the policy that is used to set the permissions boundary for the user.

Reply:
Changed